src/01-dinosaur-facts.js

```diff
@@ -71,7 +135,54 @@ function getDinosaurDescription(dinosaurs, id) {}
  *  getDinosaursAliveMya(dinosaurs, 65, "unknown-key");
  *  //> ["WHQcpcOj0G"]
  */
-function getDinosaursAliveMya(dinosaurs, mya, key) {}
+
+
+function getDinosaursAliveMya(dinosaurs, mya, key) {
+
+  //returning array of dinos that were alive based on mya
+  // if key is provided, return key value
+  //else, return ID if key is not provided
+  //if dino has single value for mya, mya === value or -1 from the number in the array [mya]
+
+  let newArr = [];
+  
+  for (let dino of dinosaurs) {
+    if(dino.mya.length === 1){ // single value
```

Comment:
So your comment has the right idea but your code isnt representing what your comment says. You have dino.mya[0] >= mya instead of 'if value is equal'